src/Phing/Project.php

```diff
@@ -863,13 +864,7 @@ public function resolveFile(string $fileName, File $rootDir = null): File
      */
     public static function toBoolean($s)
```

Comment:
@jawira feels good but after short investigation I found some locations where the `Project::toBoolean` is also used.
See:
```
Method
    toBoolean
Found usages  (9 usages found)
    Method call  (9 usages found)
        phing  (9 usages found)
            src  (9 usages found)
                Phing  (9 usages found)
                    Filter  (3 usages found)
                        LineContains.php  (1 usage found)
                            LineContains  (1 usage found)
                                initialize  (1 usage found)
                                    265 $this->setNegate(Project::toBoolean($param->getValue()));
                        LineContainsRegexp.php  (2 usages found)
                            LineContainsRegexp  (2 usages found)
                                initialize  (2 usages found)
                                    270 $this->setNegate(Project::toBoolean($params[$i]->getValue()));
                                    272 $this->setCaseSensitive(Project::toBoolean($params[$i]->getValue()));
                    Task  (2 usages found)
                        System  (2 usages found)
                            Condition  (2 usages found)
                                IsPropertyFalseCondition.php  (1 usage found)
                                    IsPropertyFalseCondition  (1 usage found)
                                        evaluate  (1 usage found)
                                            60 return null === $value || !Project::toBoolean($value);
                                IsPropertyTrueCondition.php  (1 usage found)
                                    IsPropertyTrueCondition  (1 usage found)
                                        evaluate  (1 usage found)
                                            57 return null !== $value && Project::toBoolean($value);
                    Type  (4 usages found)
                        Selector  (4 usages found)
                            ContainsRegexpSelector.php  (2 usages found)
                                ContainsRegexpSelector  (2 usages found)
                                    setParameters  (2 usages found)
                                        136 $this->setCasesensitive(Project::toBoolean($parameters[$i]->getValue()));
                                        141 $this->setMultiLine(Project::toBoolean($parameters[$i]->getValue()));
                            FilenameSelector.php  (2 usages found)
                                FilenameSelector  (2 usages found)
                                    setParameters  (2 usages found)
                                        149 $this->setCasesensitive(Project::toBoolean($parameters[$i]->getValue()));
                                        154 $this->setNegate(Project::toBoolean($parameters[$i]->getValue()));

```
Please change to the StringHelper on those locations as well and annotate the `Project::toBoolean` as deprecated.

Reply:
Done!